### PR TITLE
Enable Travis CI

### PR DESCRIPTION
**Pull request by raveit65**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    - libxklavier
+    - make
+    - mate-common
+    - which
+
+  debian:
+    # Useful URL: https://github.com/mate-desktop/debian-packages
+    # Useful URL: https://salsa.debian.org/debian-mate-team/libmatekbd
+    - debhelper
+    - dpkg-dev
```
[the remark this answers is not in the document]
Comment (raveit65) [reply .travis.yml]: Ok, i will remove it from templates at mate-dev-scripts too.
I guess `which` is always needed for every repo for archlinux?

[the remark this answers is not in the document]
Comment (raveit65) [reply .travis.yml]: error, debhelper and dpkg-dev aren't from your template.

[the remark this answers is not in the document]
Comment (yetist) [reply .travis.yml]: Yes, who requires `mate-common`, who need `which` on archlinux.

[the remark this answers is not in the document]
Comment (yetist) [reply .travis.yml]: I guess you copied them from the debian's control file, next time ignore them is better.

**Comment by raveit65** [comment]
PR is rebased.

**Comment by yetist** [comment]
ok, after build finished, you can merge it.

**Pull request merged**